bedrock/settings/base.py

```diff
@@ -1183,7 +1183,9 @@ def get_app_name(hostname):
 
 GA_ACCOUNT_CODE = ""  # DELETE ME: Deprecated?
 GTM_CONTAINER_ID = config("GTM_CONTAINER_ID", default="")  # NB: Not used in all Bedrock modes (Mozorg only).
+# Pocket mode will be running both GA UA and GA4 for a while going forward
 GOOGLE_ANALYTICS_ID = config("GOOGLE_ANALYTICS_ID", default="")  # NB: Not used in all Bedrock modes (Pocket only).
+GOOGLE_ANALYTICS_GTM_ID = config("GOOGLE_ANALYTICS_GTM_ID", default="")  # Not used in all Bedrock modes (Pocket only).
```

Comment:
This looks like it's a second env var for loading the GTM ID. If I'm wrong and both things aren't for Google Tag Manager, ignore me. But if I'm right I suggest instead you re-use `GTM_CONTAINER_ID` for Pocket mode too (adjusting comments appropriately), then we have just one GA slot and one GTM slot.

Either way, in terms of setting a GTM ID for Pocket mode, that needs to happen in www-config: add it just above here in www-config https://github.com/mozmeao/www-config/blob/main/iowa-a/bedrock-prod/pocket-deploy.yaml#L116 (that's prod, but you may also want to add it for dev and stage too, maybe?)

Reply:
makes sense, thanks @stevejalim